cves/kernel/CVE-2017-5548.yml

```diff
@@ -322,8 +338,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: True
+  note: Two people signed off on this commit as well as a cc to stable@vger.kernal.org.
```

Comment:
Who were the people that signed off on this?  Was one of them the original developer for the commit?